docs/graphql/manual/mutations/insert.rst

```diff
@@ -241,33 +245,51 @@ in the response
         affected_rows
         returning {
           id
-          title
-          author {
-            id
-            name
+          name
+          address {
+            location
+          }
+          articles {
+            title
           }
         }
       }
     }
   :response:
     {
       "data": {
-        "insert_article": {
-          "affected_rows": 2,
+        "insert_authors": {
+          "affected_rows": 3,
           "returning": [
             {
-              "id": 21,
-              "title": "Article 1",
-              "author": {
-                "id": 11,
-                "name": "Cory"
-              }
+              "address": {
+                "location": "San Francisco"
+              },
+              "name": "John",
+              "articles": [
+                {
+                  "title": "GraphQL Guide"
+                }
+              ],
+              "id": 11
             }
           ]
         }
       }
     }
 
+**How it works**
+
+Let's say an ``author`` has an ``object relationship`` called ``address`` to the ``addresses`` table and an ``array relationship`` called ``articles`` to the ``articles`` table.
```

Comment:
We can move "Let's say an ... . We want to insert an author object along with its address and one article" to before the example so that there is more context?